tests/integration/test.basics.js

```diff
@@ -73,6 +73,21 @@ adapters.forEach(function (adapter) {
       return new PouchDB(dbs.name).destroy({});
     });
 
+    it('4339 throw useful error if method called on stale instance', function (done) {
+      var db = new PouchDB(dbs.name);
+
+      db.put({
+        _id: 'cleanTest'
+      }).then(function () {
+        return db.destroy();
+      }).then(function () {
+        return db.get('cleanTest');
+      }).catch(function (err) {
+        should.equal(err instanceof Error, true, 'should be an error');
+        done();
+      });
+    });
+
```

Comment:
Since this is a promise based test I think we want to `return db.put` and not use `done` unless I am mistaken about something